hack/release-tools/tag-release.sh

```diff
@@ -26,7 +41,7 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 # Parse out the branch we're on so we can switch back to it at the end of a dry-run, where we delete the tag. Requires git v1.8.1+
-original_branch=$(git symbolic-ref --short HEAD)
+upstreamal_branch=$(git symbolic-ref --short HEAD)
```

Comment:
Minor thing, but I think this variable should be `upstream_branch`?

Reply:
argh, good catch